Documentation/doc-github-rules.md

```diff
@@ -0,0 +1,87 @@
+
+If you use the Github desktop app, see [Cloning and forking repositories from GitHub Desktop](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/adding-and-cloning-repositories/cloning-and-forking-repositories-from-github-desktop). 
+
+This approach is useful if you want to 
+
+- Fix a pubished artifact.
+- Create a new artifact.
+- Improve overall readibility, usefulness, and quality.
+   
+In this case, it's good practice to regularly sync your fork with the original repository. 
+For more information, see [Syncing a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/working-with-forks/syncing-a-fork).
+
+### Create a pull request (PR) in the original repository from a fork
+
+You can make any changes to a fork, including making branches and opening pull requests. 
+If you want to contribute to the original repository, you can send a request to the original author to pull your fork into their repository by submitting a pull request.
+
+For more information, see [Creating a pull request from a fork](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request-from-a-fork).
+
+If you use the Github desktop app, see [Creating an issue or pull request](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request). 
+
+As the author, you submit the PR to a review phase. The PR can be approved or rejected. If approved, the proposed changes are merged into the original repository by the people that have the ability to do so.
+
+## Create pull request (PR) in the original repository 
+
+You use a pull request (PR) to propose changes to a repository. A correct practice is to create a branch off the original repository and then create a PR based on this branch. You must have write access to the original repository. So this approach is for internal contributors. 
+
+As the author, you submit the PR to a review phase. The PR can be approved or rejected. If approved, the proposed changes are merged into the original repository by the people that have the ability to do so.
+
+For more information, see [Creating a pull request](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request). 
+
+If you use the Github desktop app, see [Creating an issue or pull request](https://docs.github.com/en/desktop/contributing-and-collaborating-using-github-desktop/working-with-your-remote-repository-on-github-or-github-enterprise/creating-an-issue-or-pull-request). 
+
+
+## Tools
+
+- [GitHub desktop](https://desktop.github.com/)
+- [Visual studio code](https://code.visualstudio.com/)
+  - [Gitlens](https://marketplace.visualstudio.com/items?itemName=eamodio.gitlens)
```

Comment:
Can you add the nifty ToC extension you used?

Reply:
What do you mean? I added the TOC.

Reply:
You told us you had installed an extension into VScode which automatically updates the ToC when you save the file. Can you list the extension here? I only see the GitLens one. Thanks.

Reply:
Added link.